src/DotNetWorker.Core/FunctionsApplication.cs

```diff
@@ -64,5 +91,27 @@ public Task InvokeFunctionAsync(FunctionContext context)
                 return _functionExecutionDelegate(context);
             }
         }
+
+        public void CancelInvocation(string invocationId)
+        {
+            var invocationDetails = _functionInvocationManager.TryGetInvocationDetails(invocationId);
+            if (invocationDetails?.CancellationTokenSource is not null)
+            {
+                try
+                {
+                    invocationDetails.CancellationTokenSource.Cancel();
+                }
+                catch (Exception)
+                {
+                    // what should we do here?
```

Comment:
Thoughts on if we need to handle this here i.e. if the cancellation token source has already been disposed